src/storage/src/table/cell_based_table.rs

```diff
@@ -456,3 +464,69 @@ impl<S: StateStore> TableIter for CellBasedTableRowIter<S> {
         }
     }
 }
+
+/// "`CellBasedTableStreamingIter`" used for streaming executor, and will not wait epoch
+pub struct CellBasedTableStreamingIter<S: StateStore> {
+    /// An iterator that returns raw bytes from storage.
+    iter: StripPrefixIterator<S::Iter>,
+    /// Cell-based row deserializer
+    cell_based_row_deserializer: CellBasedRowDeserializer,
+    pub cell_based_item: Option<(Vec<u8>, Row)>,
```

Comment:
Should implement Rust-style iterator for `CellBasedTableStreamingIter`. `next` should always return a value instead of storing it somewhere else.